pydantic/_internal/_generate_schema.py

```diff
@@ -1897,7 +1901,11 @@ def _callable_schema(self, function: Callable[..., Any]) -> core_schema.CallSche
             if p.annotation is sig.empty:
                 annotation = typing.cast(Any, Any)
             else:
-                annotation = type_hints[name]
+                # Note: This was originally get by `_typing_extra.get_function_type_hints`,
+                #       but we switch to simply `p.annotation` to support bultins (e.g. `sorted`).
+                #       May need to revisit if anything breaks.
+                annotation = ForwardRef(p.annotation) if isinstance(p.annotation, str) else p.annotation
```

Comment:
From the `_typing_extra` module:

```suggestion
                annotation = _make_forward_ref(p.annotation) if isinstance(p.annotation, str) else p.annotation
```